ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -40,6 +40,14 @@ ActivityNotificationBase {
             colorId: Utils.colorIdForPubkey(contactId)
             colorHash: Utils.getColorHashAsJson(contactId, sender.isEnsVerified)
         }
+        contentType: notification && notification.message ? notification.message.contentType : StatusMessage.ContentType.Unknown
+        album: notification && notification.message ? notification.message.albumMessageImages.split(" ") : []
+        albumCount: notification && notification.message ? notification.message.albumImagesCount : 0
+        messageContent: notification && notification.message ? notification.message.messageImage : ""
+    }
+
+    onNotificationChanged: {
+        print("LALALA", notification.message.albumMessageImages, notification.message.albumImagesCount)
```

Comment:
Debug leftover?

Reply:
oh... I'm searching it every time but search doesn't helps :(

Reply:
I use `!!!` personally :D